### ISSUE TITLE
Does this support ARM platform

### ISSUE THREAD

**Issue by kaloho** [WuBingzheng/memleax#46]
Hi WuBingzheng :

When I try to cross compile on ARM Platform, and the environment variables are as follows:

CC=arm-marvell-linux-gnueabi-gcc

I will get many compilation errors, does this support ARM platform ? thanks.

regards,
Kaloho

**Comment by kaloho** [comment]
@WuBingzheng :
Please could you tell me how to execute on ARM platform ? thanks

**Comment by WuBingzheng** [comment]
No, it does not support ARM. Because it use X86-64 instructions to debug.

I recommend to use [libleak](https://github.com/WuBingzheng/libleak) which uses high level libraries to debug.
I do not test it on ARM, but I think it may run on ARM.
